custom_components/tuya_local/devices/agl_ultramagic_lock.yaml

```diff
@@ -118,3 +118,90 @@ entities:
           min: 0
           max: 100
         unit: s
+
+  - entity: number
+    category: config
+    name: Record tag ID
+    icon: "mdi:nfc-variant"
+    dps:
+      - id: 101
+        name: value
+        type: integer
+        range:
+          min: 0
+          max: 2000
+  - entity: number
+    category: config
+    name: Delete tag ID
+    icon: "mdi:nfc-variant-off"
+    dps:
+      - id: 110
+        name: value
+        type: integer
+        range:
+          min: 0
+          max: 100000
```

Comment:
It seems a bit strange to have a different range for the tag IDs to be deleted than the range for adding. Where did you get these ranges from?